pkg/errors/errors.go

```diff
@@ -422,6 +427,10 @@ func (e *ServiceError) IsServiceAccountNotFound() bool {
 	return e.Code == ServiceAccountNotFound("").Code
 }
 
+func (e *ServiceError) IsMaxLimitForServiceAccountReached() bool  {
+	return e.Code == MaxLimitForServiceAccountReached("").Code
```

Comment:
I think
```suggestion
	return e.Code == ErrorMaxLimitForServiceAccountsReached
```
would be preferable since avoids an useless instantiation. WDYT?

Reply:
make sense. added